src/Sql/Sql.Test/ScenarioTests/VulnerabilityAssessmentTests.cs

```diff
@@ -52,7 +52,7 @@ public void VulnerabilityAssessmentServerSettingsTest()
 
         #region Policy Tests
 
-        [Fact]
+        [Fact(Skip = "Too long")]
```

Comment:
We need to ge a positive confirmation from the test owner (or other knowledgeable dev) that this behavior has not regressed.